PodcastGenerator/admin/episodes_list.php

```diff
@@ -0,0 +1,86 @@
+<?php
+############################################################
+# PODCAST GENERATOR
+#
+# Created by Alberto Betella and Emil Engler
+# http://www.podcastgenerator.net
+# 
+# This is Free Software released under the GNU/GPL License.
+############################################################
+require 'checkLogin.php';
+require '../core/include_admin.php';
+
+//$episodes = getEpisodes(null, $config, true);
+
+// reads all episodes into an array
+// and yes, I know this does basically what the other getEpisodes function does...
+function getEpisodeArray(){
```

Comment:
Per https://github.com/PodcastGenerator/PodcastGenerator/pull/374#pullrequestreview-579086708 could you update the original `getEpisodes()` function and make use of it here? Thanks.